### DISCUSSION TITLE
How to fetch the Github Enterprise Avatar?

### DISCUSSION THREAD

**Discussion by christopher-wilke** [octokit/octokit.js#2061] [Q&A]
When I try to use the `avatarUrl` in the `src `attribute of the `image ` tag, I get the following error:

Cross-Origin Read Blocking (CORB) blocked cross-origin response https://git.xyz.com/login?return... with MIME type text/html. See https://www.chromestatus.com/feature/5629709824032768 for more details.

**Comment by gr2m** [comment]
If you retrieve a user object, you have a `avatar_url` property, which is the URL to the avatar. There is no API endpoint to download the binary contents of the image though, and getting the image is blocked by CORB then there is nothing you can do about it from browsers. You'll have to build a server component that acts like proxy
